lib/sinon/stub.js

```diff
@@ -104,10 +104,10 @@ var proto = {
             return getCurrentBehavior(fnStub).invoke(this, arguments);
         };
 
-        functionStub.id = "stub#" + uuid++;
         var orig = functionStub;
         functionStub = spy.create(functionStub, stubLength);
         functionStub.func = orig;
+        functionStub.id = "stub#" + uuid++;
```

Comment:
Uh, sorry. Commited this by accident. I noticed that the generated `stub#${uuid}` id property is replaced by `spy.create` with `spy#{uuid}`. This change fixes it, but it's not really part of this PR. Shall I revert or just leave it?

Reply:
Maybe just put it into it's own commit?